### ISSUE TITLE
Adding Profile Management onto Auth

### ISSUE THREAD

**Issue by cz2246** [robot-monkey/final-project#7]
Let's start merging from adding profiles onto the Auth. As of now, Auth is working with hard-coded profiles. Joe and I can work on adding what Joe has done onto the my file on the project page.

**Comment by andrewolo** [comment]
Ok cool. Then we do product and catalog

**Comment by cz2246** [comment]
Joe, if you are currently too busy, we can somehow start with product and catalog?

**Comment by andrewolo** [comment]
Yea but let us know asap, no more dilly dallying. We can wrap this up asap

**Comment by cz2246** [comment]
Yes, given how the previous went we need to work on it now. If not, we need to discuss it...

**Comment by andrewolo** [comment]
I'm willing to meet physically next couple of days, I think it's worth it.
Even if only you Cem and whoever else can make it.

What do you say?

**Comment by joebeut** [comment]
hey guys,

I will be working on this all afternoon. Hope to have it done by tonight.
If not, I will be able to work on it all day tomorrow after 2:30 if you
guys want to get together.

Joe

**Comment by cz2246** [comment]
So any updates Joe?

**Comment by cz2246** [comment]
???

**Comment by andrewolo** [comment]
Yea Joe, whats the status?

**Comment by joebeut** [comment]
Hey guys,

I've really been trying to get this working. I've dedicated three different
coding sessions over the past week to trying to figure it out but I can't
seem to get angular working on my machine. I havent been able to convert my
part over from express to angular. I was set up to meet with Lev on
Wednesday but he wasn't able to make it. I am going to keep working on it
tonight, but I feel like you guys shouldnt wait for me any longer.

Sorry I haven't been more help.
Joe

**Comment by cz2246** [comment]
Were you able to run the app as is first? I can help out?

**Comment by joebeut** [comment]
Cem,

I was able to get your part working. I could not, however, get Jeff or Andrew’s part working because of MongoDB problems (at least that’s what Lev thought). I have been emailing Lev and I’ve tried a couple of things that haven’t work. I’m waiting to hear back from him.

**Comment by cz2246** [comment]
Ok, were you trying to build your part onto mine? Let me know if I can help out with anything? (Sorry for late-ish response)

**Comment by cz2246** [comment]
So guys, given this is (i think) last week or so, do we have any updates? Joe did lev respond?

**Comment by cz2246** [comment]
Anyone?
